cmd/podman/create.go

```diff
@@ -388,14 +378,62 @@ func parseCreateOpts(ctx context.Context, c *cli.Context, runtime *libpod.Runtim
 		return nil, errors.Errorf("--cpu-quota and --cpus cannot be set together")
 	}
 
-	utsMode := container.UTSMode(c.String("uts"))
-	if !cc.IsNS(string(utsMode)) && !utsMode.Valid() {
+	// Kernel Namespaces
+	var pod *libpod.Pod
+	if c.IsSet("pod") {
+		pod, err = runtime.LookupPod(c.String("pod"))
+		if err != nil {
+			return nil, err
+		}
+	}
+
+	pidModeStr := c.String("pid")
+	if !c.IsSet("pid") && pod != nil && pod.SharesPID() {
+		pidModeStr = "pod"
```

Comment:
`"pod"` has some special meaning. Can we make this a const with some explanation of its semantics? This will avoid touching many locations when having to change the string but it also documents the semantics of it.